### PR TITLE
Show my positions fix

### LINKED ISSUE #307: My positions switch on pools page not working
Screenshot 2023-02-09 at 00 11 14

reproducible on https://beta-app.basilisk.cloud

### LINKED ISSUE #312: Add no positions display when my positions switch is on and no positions are present
https://www.figma.com/file/HZt2Xm6sfGWsI3vnnnaMC9/%F0%9F%96%A5--Basilisk-v0.5-%7C-Desktop?node-id=10954%3A143109&t=6MguIsxeQyZ0fZ9O-1 here is the no-positions screen + we can adjust wording while we're at it

_Originally posted by @jak-pan in https://github.com/galacticcouncil/Basilisk-ui/issues/307#issuecomment-1424751147_

### PR DESCRIPTION
**Pull request by vkulinich-cl**
Fixes #307 
Fixes #312

### CODE REVIEW

File: src/i18n/locales/en/translations.json [hunk 1]
```diff
@@ -65,6 +65,8 @@
   "pools.header.myLocked": "My balance in Pools",
   "pools.header.myFarms": "Total balance locked in Farms",
   "pools.pool.title": "{{ poolType }} Pool",
+  "pools.emptyState.description1": "You have no active farm positions",
+  "pools.emptyState.description2": "Ape in (DYOR) and earn some rewardssss...",
```
Comment (vgantchev) [inline src/i18n/locales/en/translations.json:69]: Join a farm and earn rewards

File: src/sections/pools/PoolsPage.tsx [hunk 2]
```diff
@@ -31,8 +32,10 @@ export const PoolsPage = () => {
           <div sx={{ width: "100%", flex: "row", justify: "center" }}>
             <Spinner width={32} height={32} />
           </div>
-        ) : (
+        ) : data?.length ? (
```
Comment (tomashapl1) [inline src/sections/pools/PoolsPage.tsx:35]:
```
!!data?.length
```

**Review by tomashapl1 — CHANGES_REQUESTED** [review]
Just small change 😉

File: src/i18n/locales/en/translations.json (hunk 1 above)
Comment (vkulinich-cl) [reply src/i18n/locales/en/translations.json:69]: @vgantchev so, we are agreed on 'Join a farm and earn rewards'? or you want to make changes based on the @jak-pan comment?

File: src/i18n/locales/en/translations.json [hunk 3]
```diff
@@ -65,6 +65,8 @@
   "pools.header.myLocked": "My balance in Pools",
   "pools.header.myFarms": "Total balance locked in Farms",
   "pools.pool.title": "{{ poolType }} Pool",
+  "pools.emptyState.description1": "You have no active farm positions",
```
Comment (jak-pan) [inline src/i18n/locales/en/translations.json:68]:
```suggestion
  "pools.emptyState.description1": "You have no active positions",
```

File: src/i18n/locales/en/translations.json [hunk 4]
```diff
@@ -65,7 +65,7 @@
   "pools.header.myLocked": "My balance in Pools",
   "pools.header.myFarms": "Total balance locked in Farms",
   "pools.pool.title": "{{ poolType }} Pool",
-  "pools.emptyState.description1": "You have no active farm positions",
+  "pools.emptyState.description1": "You have no active positions",
   "pools.emptyState.description2": "Ape in (DYOR) and earn some rewardssss...",
```
Comment (jak-pan) [inline src/i18n/locales/en/translations.json]:
```suggestion
  "pools.emptyState.description2": "Provide liquidity to support the protocol",
```

Comment (jak-pan) [reply src/i18n/locales/en/translations.json]: We can change this wording later. We should have a warning / explanation everywhere about the risks of LPing. Until then I would not go aggresive.

**Review by jak-pan — APPROVED** [review]

**Comment by jak-pan** [comment]
LGTM

File: src/i18n/locales/en/translations.json [hunk 5]
```diff
@@ -65,6 +65,8 @@
   "pools.header.myLocked": "My balance in Pools",
   "pools.header.myFarms": "Total balance locked in Farms",
   "pools.pool.title": "{{ poolType }} Pool",
+  "pools.emptyState.description1": "You have no active positions",
```
Comment (tomashapl1) [inline src/i18n/locales/en/translations.json:68]: I would prefer naming like:

`pools.emptyState.title` and `pools.emptyState.description`

**Review by tomashapl1 — CHANGES_REQUESTED** [review]

File: src/i18n/locales/en/translations.json (hunk 5 above)
Comment (vkulinich-cl) [reply src/i18n/locales/en/translations.json:68]: but both of them are description strings. There is no name, just description

Comment (tomashapl1) [reply src/i18n/locales/en/translations.json:68]: OK, so go with `<br />`

```
You have no active positions <br /> Provide liquidity to support the protocol
```

**Review by tomashapl1 — APPROVED** [review]

**Pull request merged**